hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -167,6 +167,14 @@ def _get_tags(data):
 # Begin function definitions
 ############################
 
+
+def _execute_shell_command(cmd):
+    '''
+    This function will execute passed command in /bin/shell
+    '''
+    return __salt__['cmd.run'](cmd, python_shell=True, shell='/bin/bash', ignore_retcode=True)
```

Comment:
Source of many troubles. I recommend against using arbitrary shell commands do underlying work.

Reply:
Yep. python_shell=True should be avoided wherever possible.

Reply:
Sure, will open a new ticket and will plan to handle this issue.